### PR TITLE
Move to new COPR

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by jpopelka** [comment]
http://nooooooooooooooo.com

**Comment by msrb** [comment]
:P

**Comment by msrb** [comment]
[test]

**Comment by msrb** [comment]
Meh... still old credentials in CI.

**Comment by msrb** [comment]
[test]

File: cico_setup.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ create_copr_config() {
     cat > ~/.config/copr <<EOF
 [copr-cli]
 login = ${COPR_LOGIN_MERCATOR}
-username = jpopelka
+username = msrb
```
Comment (jpopelka) [inline cico_setup.sh]: You could also update the expiration date in above comment.

**Comment by msrb** [comment]
[test]

**Pull request merged**